org/cloudfoundry.yml

```diff
@@ -2388,7 +2388,7 @@ orgs:
         members: []
         privacy: closed
         repos:
-          claw: admin
+          CLAW: admin
```

Comment:
Our working theory is that case sensitivity is breaking cloudfoundry/community synchronization